src/cli/global/install.rs

```diff
@@ -227,12 +219,12 @@ async fn setup_environment(
         }
     }
 
-    if project.environment_in_sync(env_name).await? {
+    if project.environment_in_sync_internal(env_name, true).await? {
```

Comment:
Why is that specific method `internal`?

Reply:
It's basically an extra layer of indirection, which is not exposed directly